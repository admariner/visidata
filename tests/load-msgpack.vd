sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/test.msgpack	o	open input in VisiData
test		0	expand-cols-depth	0	gz(	expand all visible columns of containers to given depth (0=fully)
test			columns-sheet		C	open Columns Sheet
test_columns	name		sort-asc		[	sort ascending by current column
test_columns			quit-sheet		q	quit current sheet
