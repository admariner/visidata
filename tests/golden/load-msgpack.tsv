:@computed_region_cbhk_fwbd	:@computed_region_nnqa_25f4	fall	geolocation.coordinates[0]	geolocation.coordinates[1]	geolocation.type	id	mass	name	nametype	recclass	reclat	reclong	year
		Fell	6.08	50.77	Point	1	21	Aachen	Valid	L5	50.775000	6.083330	1880-01-01T00:00:00.000
		Fell	10.23	56.18	Point	2	720	Aarhus	Valid	H6	56.183330	10.233330	1951-01-01T00:00:00.000
